MAINTAINERS.md

```diff
@@ -1,14 +1,17 @@
-## Maintainer
+## Maintainers
 
-They are approvers who have shown good technical judgement in feature design/development in the past. Has overall knowledge of the project and features in the project. These role is for project managers who are expected to manage the repository without access to sensitive or destructive actions.
+Maintainers are approvers who have shown good technical judgement in guiding feature design & development, have displayed overall knowledge of the project and features in the project, and are nuturing and receptive to everyone in the community.
 
-| Maintainer  | GitHub ID                                   | Affiliation               |
-| ----------- | ------------------------------------------- | ------------------------- |
-| JimBugwadia | [@JimBugwadia](https://github.com/JimBugwadia) | Nirmata Inc.              |
-| Realshuting | [@realshuting](https://github.com/realshuting) | Nirmata Inc.              |
-| Chip zoller | [@chipzoller](https://github.com/chipzoller)  | dell                      |
-| Marcel Mue  | [@MarcelMue](https://github.com/MarcelMue)   | Giant Swarm GmbH          |
-| Treydock    | [@treydock](https://github.com/treydock)    | Ohio Supercomputer Center |
+| Maintainer            | GitHub ID                                              | Affiliation               |
+| --------------------- | ------------------------------------------------------ | ------------------------- |
+| Jim Bugwadia          | [@JimBugwadia](https://github.com/JimBugwadia)         | Nirmata                   |
+| Shuting Zhao          | [@realshuting](https://github.com/realshuting)         | Nirmata                   |
+| Chip Zoller           | [@chipzoller](https://github.com/chipzoller)           | Dell                      |
+| Marcel Muller         | [@MarcelMue](https://github.com/MarcelMue)             | Giant Swarm GmbH          |
```

Comment:
Marcel Müller

Reply:
Was trying to get the accent characters....how did you get these? The only way I could find is copy and paste.

Reply:
If you're on Mac, it's (Alt + u), u